package.json

```diff
@@ -37,12 +37,20 @@
         "lint:fix:md": "markdownlint --fix  \"**/*.md\"",
         "lint:scss": "stylelint \"**/*.scss\"",
         "lint:fix:scss": "stylelint \"**/*.scss\" --fix",
+        "fmt": "prettier --write \"**/*.+(js|json|scss)\"",
+        "fmt:check": "prettier --check \"**/*.+(js|json|scss)\"",
```

Comment:
@mdjermanovic we run prettier on markdowns like `readme.md` too in rewrite repo? Should we do the same here? and ignore all the website markdown contents through prettier ignore?

Reply:
I think it's fine to start with .js, .json, and .scss files only and evaluate using prettier on .md files as well at some later point.

Reply:
note here, here are changes i had to made to markdownlint file to make it in sync with prettier and not throw errors.
<img width="670" alt="Screenshot 2024-06-18 at 7 11 41 PM" src="https://github.com/eslint/eslint.org/assets/49820024/f963c039-d815-489c-bbca-77373fbd9034">

Reply:
please let me know if you want to proceed with md formatting and any ones i can ignore, should be quick.

Reply:
I'd start with just js/json/scss and consider enabling prettier on other extensions later.